package.json

```diff
@@ -19,6 +19,8 @@
         "lint:diagnostics": "ttsc -p tsconfig.diagnostics.json",
         "test": "jest --passWithNoTests",
         "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+        "test:e2e": "jest --forceExit --config ./jest.config.e2e.ts",
```

Comment:
a tu ten --forceExit to dobry pomysł? może lepiej wiedzieć, że jakiś resource jest niezamkniety